composer.json

```diff
@@ -3,7 +3,7 @@
     "type": "typo3-cms-extension",
     "version": "1.7.0",
     "require": {
-        "typo3/cms-core": ">=9 <12"
+        "typo3/cms-core": "^11.5.26 <12"
```

Comment:
I personally wouldn't change the dependency here (and remove two versions of compatibility) but would just introduce an own function to get the PageRepository:

```
    protected function getPageTreeRepository(): PageTreeRepository
    {
        if (method_exists(\TYPO3\CMS\Backend\Controller\Page\TreeController::class, 'getPageTreeRepository')) {
            return parent::getPageTreeRepository();
        }

        return $this->pageTreeRepository;
    }
```

Reply:
Great idea, thank you